Tasks/skip.js

```diff
@@ -1,17 +1,21 @@
+'use strict';
+
 // Remove values from array
 
-Skip =(T,...Remove)=> {
-x = 0;
-for (C of T) {
-for (X of Remove) {
-if (C === X) {
-  T.splice(x, 1);
+const skip = (array, ...charsToRemove) => {
+  let removedChar = 0;
+  for (const selectedChars of array) {
+    for (const selectedCharsToRemove of charsToRemove) {
```

Comment:
Try to avoid nested loops

Reply:
Got it! Tried to remove one loop and use some methods instead